src/app/components/VerifyingIfAnswerIsCorrect.js

```diff
@@ -0,0 +1,11 @@
+
+export default function VerifyingAnswerIsCorrect() {
+  const userAnswerIsCorrect = { correct: "" };
+  function verifyingAnswer(userAnswer, rightAnswer) {
+    userAnswer === rightAnswer ? userAnswerIsCorrect.correct = true : userAnswerIsCorrect.correct = false;
+
+    console.log(userAnswerIsCorrect);
```

Comment:
It's not supposed to be in the production code.

Reply:
It was removed